index_service/locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,307 @@
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+#
+# mabe <beermann@qabel.de>, 2016.
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-09-26 09:07+0200\n"
+"PO-Revision-Date: 2016-09-26 09:07+0100\n"
+"Last-Translator: mabe <beermann@qabel.de>\n"
+"Language-Team: English <kde-i18n-doc@kde.org>\n"
+"Language: en_GB\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Lokalize 2.0\n"
+
+#: admin.py:6
+msgid "Index"
+msgstr ""
+
+#: admin.py:7 admin.py:8
+msgid "Qabel Index Admin"
+msgstr ""
+
+#: models.py:57
+msgid "E-Mail address"
+msgstr "E-Mail-Addresse"
+
+#: models.py:58
+msgid "Phone number"
+msgstr "Telefonnummer"
+
+#: templates/codeentry.html:11
+msgid "Enter the confirmation code you received to proceed"
+msgstr "Geben Sie einen Bestätigungscode ein, um fortzufahren"
```

Comment:
Yes, this page was created to cater the possibility that we need to shorten the URL further, since the current SMS template is almost at the maximum length.